examples/make_bg_cube_models_true-reco_test.py

```diff
@@ -0,0 +1,217 @@
+"""
+Script to produce true and reco background cube models.
+
+The script uses the same model (except for absolute normalization) to
+produce a true cube bg model and a reco cube bg model. The models can
+be used to test the cube bg model production and can be compared to
+each other using the plot_bg_cube_model_comparison.py example script.
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals) # python 2 as python 3
+import os
+import numpy as np
+from astropy.units import Quantity
+from astropy.coordinates import Angle
+from astropy.time import Time
+from gammapy.datasets import make_test_bg_cube_model, make_test_dataset
+from gammapy.obs import (DataStore, ObservationGroups,
+                         ObservationGroupAxis)
+from gammapy.background import make_bg_cube_model
+
+
+#TEST = True # create a small dataset
+TEST = False # create a large dataset (slow)
+
+
+# using values from scripts.group_observations
+AZ_RANGE = Angle([90, 270], 'degree')
+ALT_RANGE = Angle([72, 90], 'degree')
+GROUP_ID = 27
+OUTDIR = 'bg_cube_models'
+OVERWRITE = False
+
+
+def create_dummy_observation_grouping():
+    """Define dummy observation grouping."""
+
+    altitude_edges = ALT_RANGE
```

Comment:
I think this function can be shortened by half:
```python
def create_dummy_observation_grouping():
    alt_ax = ObservationGroupAxis('ALT', ALT_RANGE, 'bin_edges')
    az_ax = ObservationGroupAxis('AZ', AZ_RANGE, 'bin_edges')
    obs_groups = ObservationGroups([alt_ax, az_ax])
    obs_groups.obs_groups_table['GROUP_ID'][0] = GROUP_ID

    return obs_groups
```

The following functions can also be simplified / improved.

Just in case you say "this is just an example and not worth the effort of code review / polishing", I'd say "especially for examples it's important to make them nice and clean" ... it's how users / co-developers learn and what determines whether they like Gammapy or now.